src/scheduler/fifo.cpp

```diff
@@ -1510,7 +1511,9 @@ run_update_resresv(status *policy, int pbs_sd, server_info *sinfo,
 	pbs_errno = PBSE_NONE;
 	if (resresv->is_job && resresv->job->is_suspended) {
 		if (pbs_sd != SIMULATE_SD) {
-			pbsrc = pbs_sigjob(get_svr_inst_fd(pbs_sd, resresv->job->svr_inst_id), resresv->name, const_cast<char *>("resume"), NULL);
+			pbsrc = pbs_sigjob(get_svr_inst_fd(pbs_sd, resresv->job->svr_inst_id), resresv->name,
+					   const_cast<char *>("resume"), NULL);
```

Comment:
When you rebase, you'll notice you'll need to do const_cast<char *>(resresv->name.c_str).  resresv->name is now a std::string.